system/Filters/DebugToolbar.php

```diff
@@ -41,5 +42,7 @@ public function before(RequestInterface $request, $arguments = null)
     public function after(RequestInterface $request, ResponseInterface $response, $arguments = null)
     {
         service('toolbar')->prepare($request, $response);
+
+        return $response;
```

Comment:
Should this be `null`, too?

Reply:
do we have specific verification whether it should return response or not? I return response because response changed

Reply:
I'm not sure. Let's just target `4.6` as return changed. Let's wait for other reviews on this.

Reply:
just changed target and rebased against 4.6 branch

Reply:
Filters are pretty well explained: https://codeigniter.com/user_guide/incoming/filters.html#before-filters

And from what I see the description is up to date: https://github.com/codeigniter4/CodeIgniter4/blob/develop/system/Filters/Filters.php#L194

I think that returning the `$response` should be reserved for cases when we create a new instance of it. Any changes made to the `$response` object within the filter will be reflected outside of it without the need to return and replace the object.

IMO in this case we should just return `null`.

Reply:
ok, updated to returns null https://github.com/codeigniter4/CodeIgniter4/pull/9311/commits/00b3b1bbd75eef1e2ff84e715d47aa6709019ba9